skipper_test.go

```diff
@@ -544,3 +544,32 @@ func TestDataClients(t *testing.T) {
 
 	sigs <- syscall.SIGTERM
 }
+
+func TestTracerInstanceTakesPrecedenceOverTracingOptions(t *testing.T) {
```

Comment:
Not sure if we need full integration test here, maybe lets make it similar to TestOptionsFilterRegistry

Reply:
Will amend this as well.